js/src/product-feed/product-statistics/create-campaign-notice.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Notice } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import AddPaidCampaignButton from '.~/components/paid-ads/add-paid-campaign-button';
+import useMCProductStatistics from '.~/hooks/useMCProductStatistics';
+import useAdsCampaigns from '.~/hooks/useAdsCampaigns';
+
+const CreateCampaignNotice = () => {
+	const { hasFinishedResolution: productsResolution, data: products } =
+		useMCProductStatistics();
+	const { loaded: campaignsLoaded, data: campaigns } = useAdsCampaigns();
+
+	const isLoading =
+		! productsResolution || products?.loading || ! campaignsLoaded;
+
+	const inValidData =
```

Comment:
```suggestion
	const invalidData =
```

Reply:
Removed the variable as your next comment made sense to me